.github/workflows/linux.yml

```diff
@@ -93,16 +95,16 @@ jobs:
         mkdir build
         cd build
         cmake -D CMAKE_BUILD_TYPE=Debug \
-              -D DEAL_II_CXX_FLAGS='-Werror -std=c++17' \
+              -D DEAL_II_CXX_FLAGS='-Werror -std=c++20' \
               -D DEAL_II_EARLY_DEPRECATIONS=ON \
-              -D DEAL_II_WITH_CGAL="ON" \
+              -D DEAL_II_COMPONENT_EXAMPLES="OFF" \
```

Comment:
Do you know why we are turning examples off?

Reply:
Beats me. @tjhei's plan is to convert all example steps into full fledged tests under `tests/examples` anyway over the next weeks so I simply left it disabled.